pkg/api/dashboard.go

```diff
@@ -122,7 +121,8 @@ func (hs *HTTPServer) GetDashboard(c *models.ReqContext) response.Response {
 		meta.FolderUrl = query.Result.GetUrl()
 	}
 
-	provisioningData, err := dashboards.NewProvisioningService().GetProvisionedDashboardDataByDashboardID(dash.Id)
+	svc := dashboards.NewService(hs.SQLStore, hs.SQLStore, hs.DataService)
```

Comment:
Nit, providing sqlstore as both first and second argument looks a bit strange. But inspecting the function signature it makes sense.

Reply:
@marefr Yeah, it has to be that way when using two different interfaces, the alternative would be to join into one.

Reply:
It's fine as is.